notebooks/evaluation/baseline_evaluation.ipynb

```diff
@@ -0,0 +1,183 @@
+     ]
+    }
+   ],
+   "source": [
+    "# create the references and baseline questions\n",
+    "refs = []\n",
+    "baseline_questions = []\n",
+    "for i, row in df_test.iterrows():\n",
+    "    refs.append((i, [row['Question 1'], '' if pd.isna(row['Question 2']) else row['Question 2'], '' if pd.isna(row['Question 3']) else row['Question 3']]))\n",
```

Comment:
You can use a list comprehension to do this, there is no need to add empty strings. 
`refs.append([row[col] for col in["Question 1", "Question 2", "Question 3"] if not pd.isna(row[col])])`

Reply:
I did it based on what Leon told me.